frontend/src/app/analytics/Analytics.tsx

```diff
@@ -199,82 +184,99 @@ export const Analytics = (props: Props) => {
               {dateRange === "custom" && (
                 <div className="grid-row grid-gap margin-top-2">
                   <div className="grid-col-4">
-                    <DatePicker
-                      name="startDate"
-                      label="Begin"
-                      onChange={(date?: string) => {
-                        if (date && date.length === 10) {
```

Comment:
Because we don't have this length check anymore, one new behavior I noticed is that we are getting the stats back as we type in the date input. Not sure which behavior we would like to keep. 🤔

Reply:
(BLOCKING) Question before approving^

Reply:
Before if you wanted to change the date (type it in), it would clear out the box every time (making it impossible to actually just change one value in the date without the date picker) and would only allow the 10 digit dates so something like 9/13/2022 would not be valid and you would have to type the 0 to make sure all 10 characters were there. 


https://user-images.githubusercontent.com/10108172/191570671-ec5fbb74-b1e9-4293-9db3-fc3eb15f25f7.mp4



If we want to prevent an instant search I could put some debouncer or something like that to wait a couple of seconds for the user to be done typing, but personally I don't think that will impact the user experience/performance significantly anyways.

Reply:
Got it as long as we are fine with this behavior that works for me! thank you, boban!

Reply:
I don't have concerns about user experience with this change, I think it's better, but I do think we'll want to check the performance of the query against organizations with many tests. (As I recall the query is pretty expensive to run, it amounts to SELECT * FROM TEST_EVENT WHERE ORG_ID = $ID and I'm not sure we have an index on org_id). 

I'm not sure if we still have any test orgs with huge volumes of test events...there are a decent number in stg, but we can't deploy there without merging to `main` first. Maybe a case for Alis's DB anonymizer script?